src/scheduler_multi_app.cpp

```diff
@@ -95,21 +199,20 @@ void multi_app_sched::schedule(const job_queueT &job_queue, fast::MQTT_communica
 
 		while (true) {
 			// for all host-id of new job
-			auto marked_machines = check_membw(config);
+			const auto marked_machines = check_membw(config);
 
 			// everything fine?
 			if (marked_machines.empty()) break;
 
 			if (controller.update_supported()) {
```

Comment:
Is it important if the machines are frozen? Should we thaw them before migration? IIRC we had a discussion on first-touch in case the machine is migrated to a different slot ... but I am not sure if it really true, doesn't make sense right now.